lib/formatters/terminalLink.mjs

```diff
@@ -0,0 +1,21 @@
+import supportsHyperlinks from 'supports-hyperlinks';
+
+// ANSI escapes
+const OSC = '\u001B]';
+const BEL = '\u0007';
+const SEP = ';';
+
+/**
+ * @see https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda
+ *
+ * @param {string} text
+ * @param {string} url
+ * @returns {string}
+ */
+export default function terminalLink(text, url) {
+	if (supportsHyperlinks.stdout) {
+		return [OSC, '8', SEP, SEP, url, BEL, text, OSC, '8', SEP, SEP, BEL].join('');
+	}
+
+	return text;
+}
```

Comment:
question

Since you don't have an index.mjs for formatters, this file is only used by terminalLink.test.mjs?

Reply:
No. `terminalLink.mjs` is also used by `stringFormatter.mjs` and `verboseFormatter.mjs`.

https://github.com/stylelint/stylelint/blob/b02c39aaef66c49d93263433be8ef5508da7f0ec/lib/formatters/stringFormatter.mjs#L11

https://github.com/stylelint/stylelint/blob/b02c39aaef66c49d93263433be8ef5508da7f0ec/lib/formatters/verboseFormatter.mjs#L6

Reply:
…and stringFormatter.mjs and verboseFormatter.mjs are not used in formatters.index.mjs—which doesn't exist yet—, hence it's only actually used in terminalLink.test.mjs, correct?

Reply:
Ah, yes. I will create `formatters/index.mjs` later, but the job seems difficult since `import-lazy` is related.